_compdemos/Cromwell.md

```diff
@@ -4,18 +4,101 @@ primary_reviewers: vortexing
 
+An even more valuable benefit exists for high performance computing resources.  When Cromwell is used as an intermediary, the backend configurations (or instructions for how to send jobs to HPC resources) can be defined once for all those resources currently available to Hutch users (such as our local SLURM cluster or AWS Batch for cloud computing).  This means that individual users do not have to become SLURM or AWS Batch experts in order to be able to run jobs on them.  They instead can focus on building their workflow and analyzing their results.  
 
-More to come!  Contact Amy Paguirigan (`apaguiri`) for more information about using Cromwell at Fred Hutch.  Join the discussion in The Coop Slack in the [#cromwell-wdl channel](https://fhbig.slack.com/archives/CTFU13URJ).  
+### Reproducibility and Portability
+Beyond the benefits of abstracting some of the institution-specific details from the analysis itself, this creates a convenient side effect.  The lack of need to tailor a workflow itself to the unique combination of data storage and computing resources, means that researchers focus their time on developing workflows using the workflow language WDL.  WDL is an open source workflow description language that one can use to describe the inputs, environments and outputs of individual tasks and how they are strung together for an analysis.  By creating workflows using tasks with these features defined, it makes these workflows far more reproducible (e.g. running a given workflow on any compute infrastructure will create results that are identical), and far more portable.  In academia where collaborations and career transitions mean potential loss of time spent tailoring analyses to various institutions' infrastructure is the name of the game, this benefit holds substantial value.  
+
+### Software and Environments
+One challenge associated with reproducibility and portability that is something often undervalued in many bioinformatic analyses, is the management of computing software and environments.  As projects evolve, they often begin in a testing phase with small datasets using readily available software or packages on a desktop computer or even potentially on a shared, remote computing resource. As they scale, it often is natural to simply do X, but Y times, all using the same computing environment of the user who originally developed the workflow.  
+
+While this is common, it also creates headaches later.  Using a workflow manager like Cromwell as tool for deploying your analyses means also that you can begin your work directly with developing a WDL and running the tests via Cromwell.  Then as you develop your workflow and want to scale it to an entire analysis, you can simply edit your workflow and Cromwell will keep track of all the tasks and their interrelations.  
```

Comment:
While this is common, it also creates headaches later.  Using a workflow manager like Cromwell as a tool for deploying your analyses also means that you can begin your work directly by developing a WDL and running the tests via Cromwell.  As you develop your workflow and want to scale it to an entire analysis, you can simply edit your workflow and Cromwell will keep track of all the tasks and their interrelations.